tests/components/lcn/test_init.py

```diff
@@ -125,3 +125,24 @@ async def test_async_setup_from_configuration_yaml(hass: HomeAssistant) -> None:
         await setup_component(hass)
 
         assert async_setup_entry.await_count == 2
+
+
+@patch("homeassistant.components.lcn.PchkConnectionManager", MockPchkConnectionManager)
+async def test_migrate_1_1(hass: HomeAssistant, entry, entry_v1_1) -> None:
+    """Test migration config entry."""
+    entry_v1_1.add_to_hass(hass)
+
+    hass.config_entries.async_update_entry(
+        entry_v1_1,
+        version=1,
+        minor_version=1,
+    )
```

Comment:
We only use v1_1 here right? Should we just create it here instead of using a fixture? I find it kinda strange to use a fixture for a test and directly run async update entry to modify it